internal/errors/check.go

```diff
@@ -0,0 +1,35 @@
+package errors
+
+// Check returns retry mode for err.
+func Check(err error) (
+	statusCode int32,
```

Comment:
`Ydb.StatusIds_StatusCode`?
And need special case for -1 mapping (Unspecified looks good).